### PR TITLE
Ollie/tweak stmux

### PR DESCRIPTION
**Pull request by ob6160**
Some tweaks to the dev commands in the makefile so that we default to running as before and make the tiled view an optional extra.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,6 @@
     "tsc": "tsc --noEmit --skipLibCheck",
     "watch:server": "nodemon --inspect ./build/server.js",
     "watch": "webpack --watch --config webpack.development.js",
-    "dev": "stmux -M -e ERROR -m beep -- [ [ \"yarn watch\" .. \"yarn watch:server\" ] ]",
```
Comment (coldlink) [inline package.json:22]: removing `yarn dev` would mean the `make dev` command wouldn't work

either `yarn dev`  or `make dev` should become `yarn watch:server & yarn watch & wait`

**Review by coldlink — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (ob6160) [reply package.json:22]: oh hm, I thought I had changed that - thanks for catching

Comment (ob6160) [reply package.json:22]: fixed 👍

File: makefile [hunk 2]
```diff
@@ -25,7 +25,17 @@ build: clear clean-build install
 dev: clean-build clear install
 	$(call banner, "gateway is starting")
 	$(call log, "starting development server")
-	@yarn dev
+	@(set -a && source .env && yarn watch:server & yarn watch & wait)
```
Comment (coldlink) [inline makefile:28]: Ah! Another small change, we don't need the `set -a && source .env` anymore, as we have a webpack plugin which does that for us now. 

https://github.com/guardian/gateway/blob/60459676fd596288e48dad583a9487b6b4f754a1/webpack.config.js#L63

```suggestion
	@(yarn watch:server & yarn watch & wait)
```

Comment (ob6160) [reply makefile:28]: ah just removed this :)

**Pull request merged**